pkg/webhook/admission/builder/doc.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Package builder provides methods to build admission webhooks.
+
+For most users, we recommend to use Operations and ForType to construct a webhook.
+
+	webhook1, err := NewWebhookBuilder().
+		Mutating().
+		Operations(admissionregistrationv1beta1.Create).
+		ForType(&corev1.Pod{}).
+		WithManager(mgr).
+		Handlers(mutatingHandler11, mutatingHandler12).
+		Build()
+	if err != nil {
+		// handle error
+	}
+
+	webhook2, err := NewWebhookBuilder().
+		Validating().
+		Operations(admissionregistrationv1beta1.Create, admissionregistrationv1beta1.Update).
+		ForType(&appsv1.Deployment{}).
+		WithManager(mgr).
+		Handlers(validatingHandler21).
+		Build()
+	if err != nil {
+		// handle error
+	}
+
+Note: To build a webhook for CRD, you need to ensure the manager is
+constructed from the scheme that understands your CRD.
```

Comment:
Explain what this means more, show a code snippet of getting a manager constructed from a Scheme